patches/node/feat_initialize_asar_support.patch

```diff
@@ -1,8 +1,9 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Shelley Vohr <shelley.vohr@gmail.com>
 Date: Thu, 13 Sep 2018 08:56:07 -0700
-Subject: feat: initialize asar support
+Subject: feat_initialize_asar_support.patch
```

Comment:
this patch subject still seems wrong

Reply:
most of the Chromium patches are formatted this way! it makes it easier to find the file from the subject